governance/polls/templates/Collateral Onboarding/Proposal - Add $ilk as a new RWA Vault Type - $date_MONTH_DD,_YYYY.md

```diff
@@ -28,12 +28,12 @@ This poll allows the MakerDAO governance community to signal their support or op
 
 ### Collateral Parameters
 
-* [Stability Fee](https://community-development.makerdao.com/en/learn/governance/param-stability-fee): $stability_fee
-* [Debt Ceiling](https://community-development.makerdao.com/en/learn/governance/param-debt-ceiling): $debt_ceiling
+* [Stability Fee](https://manual.makerdao.com/parameter-index/vault-risk/param-stability-fee): $stability_fee
+* [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling): $debt_ceiling
 * Minimum Vault Collateralization Ratio: $min_vault_cr
```

Comment:
We should add a link to the 'doc' parameter here as well!

Reply:
What 'doc' parameter are you talking about? Not sure I understand what I have to do.

Reply:
This one :) https://manual.makerdao.com/parameter-index/vault-risk/param-rwa-agreement